doc/introduction/configuration.rst

```diff
@@ -52,4 +52,39 @@ Configuration options
 
 **port (int)** (*optional*)
     The port to be used when connecting to the remote service.
-    Corresponding environment variable: ``SF_API_PORT``
\ No newline at end of file
+    Corresponding environment variable: ``SF_API_PORT``
+
+Store your account
+------------------
+
+Using the :func:`~.store_account` function, a configuration file can be created easily. It only requires specifying the authentication token. Apart from that, further configuration options can be passed as keyword arguments.
+
+Default configuration
+*********************
+
+The following is an example for using ``store_account`` with defaults:
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken")
+
+where ``"MyToken"`` contains the user-specific authentication token.
+
+It is advised to execute this code snippet **only once**, separately any other Python scripts.
+
+.. note::
+
+    Using the default options will store the account in the *Strawberry Fields configuration directory* by creating a ``config.toml`` file.
+
+Configuration for a project
+***************************
+
+The following code snippet can be run in the *same directory* of a Python script or Jupyter Notebook that uses Strawberry Fields to create a configuration file *only for this project*:
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken", location="local")
+
+where ``"MyToken"`` is the user specific authentication token.
```

Comment:
Slight suggestion for the above: I would recommend shortening it (removing the two subsections), and just having one or two small examples. For additional details, the user can be redirected to the docstring for `store_account`, which should include all details about the function, plus more complicated examples.

Think of it as similar to how we do the 'quickstart' section in PennyLane. For example, the [measurements quickstart](https://pennylane.readthedocs.io/en/stable/introduction/measurements.html) briefly introduces the available functionality, with the user directed to go to the [expval docstring](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.expval.html) for more examples and details.

Reply:
Also, if you do go into the details of the project-level configuration, maybe have a motivation of why one would want to do this... e.g., is it so you can have a configuration that is different to the default?

Reply:
Still seems a bit confusing to discuss specific projects having separate configurations. Maybe we shouldn't anticipate the user's needs too much here and just state plainly how to set global and local configurations

Reply:
I've implemented the suggestion coming from @josh146 such that here we have a smaller section only with two small examples :blush:

Reply:
@josh146 I've further added examples to the docstring of `store_account`

Reply:
Agree with @co9olguy above